tools/tsp-client/src/typespec.ts

```diff
@@ -109,10 +109,11 @@ export async function compileTsp({
     for (const diagnostic of program.diagnostics) {
       const location = getSourceLocation(diagnostic.target);
       const source = location ? location.file.path : "unknown";
-      console.error(
+      Logger.error(
         `${diagnostic.severity}: ${diagnostic.code} - ${diagnostic.message} @ ${source}`,
```

Comment:
Btw typespec export functions to format diagnostics nicely as it seems here you lose information

Reply:
I see, let me take a look at that :)